config/schemas/settings.json

```diff
@@ -529,6 +530,10 @@
 				"skipBattleIntroMusic" : {
 					"type" : "boolean",
 					"default" : false
+				},
+				"infoBarCreatureManagement": {
+					"type" : "boolean",
+					"default" : false
```

Comment:
By the way - should we keep it on or off by default?

Reply:
I thought about "off" to keep compatibility with H3, same as you plan to keep mouse map dragging off

Reply:
Mouse dragging has side effect.
Map click (set path/move hero) with mouse dragging is processed on mouse *release*. This leads to visual lag since it looks like game reacts on click way than in H3 since H3 set path / move hero on mouse *press*.

Infobar change also have side effect (can't click through infobar windows), but this is very rare use-case, so probably not critical.

Reply:
did you see how HD mod resolved this issue for double click for right mouse button BTW?

Reply:
in HD mod you can click through if you target the uninteractable part (upper half), though it is not even certain if it's better to make it as in HD mod or keep it as is, since misclicks that toggle infobar may be undesired as well

Reply:
Yeah, I've seen it.
Don't think that this is significant issue - infobox is used *extremely* rarely, so likely very few players would be against such change.

So I'd rather keep current (easiest) solution for now and check if players are OK with this.

Reply:
@IvanSavenko infobox with infobox notifications can be useful more than in OH3. So, we need to be sure than army management in infobox is not interfere with infobox notifications.

Honestly, I would rather extend infobox dialog functionality than replicate a HD mod, but now I do not have spare time for hobbies(